apiserver/errors/errors.go

```diff
@@ -200,14 +237,14 @@ func singletonCode(err error) (string, bool) {
 	return code, ok
 }
 
-func singletonError(err error) (error, bool) {
+func singletonError(err error) (bool, error) {
```

Comment:
Not sure about this. I know it's the convention for error to be last, but that's for when the first return is the subject and the error is that generated in its production.

Whereas the error is the subject here.